elastic/apm-server
# Update to elastic/beats@abd6a93bf275

## Motivation/summary

Update beats. This brings in https://github.com/elastic/beats/pull/24964.

## How to test these changes

1. Install apm-server (don't just `make apm-server`, this is a shortcut for development and does _not_ build PIE)
2. Check the binary is PIE (see https://unix.stackexchange.com/questions/89211/how-to-test-whether-a-linux-binary-was-compiled-as-position-independent-code)

## Related issues

Closes #4960
